docs/ref/methods_and_combinators.rst

```diff
@@ -130,17 +130,20 @@ can be used and manipulated as below.
 
    .. versionadded:: 2.0
 
-   .. method:: optional()
+   .. method:: optional(default=None)
```

Comment:
Sphinx syntax here fails me, do I need to use `[default=None]` to indicate that it's an optional argument?

Reply:
I can't tell from the docs so including the `[ ]` for clarity.

Reply:
You actually write it just the way it appears in the sig (apart from `self` parameter). But I'll fix it up, thanks.